### DISCUSSION TITLE
Block selection should not be a mode.

### DISCUSSION THREAD

**Discussion by ANTGOMEZ** [dbeaver/dbeaver#13440] [Q&A]
Hi,

In all modern text editors I have used, Block selection (Shft-Alt-LeftMouse Drag) is not a mode like it is here in DBeaver.

Makes usage a bit awkward as I have to exit out of it before I can continue to use edit normally. And it's not standard behaviour.

Open any editor, from Outlook email, to Notepad++ to VSCode to MSWord to MSSQL, you'll see the feature for block select  is always SHFT-ALT-LeftMouseButton Drag.

If you could update it, would come inline with standard behaviour.

Thanks!

**Comment by p91paul** [comment]
That's Eclipse way of handling this, probably dbeaver is just inheriting it.
